ethpm_types/abi.py

```diff
@@ -353,7 +366,7 @@ def signature(self) -> str:
         return f"{self.name}({input_args})"
 
 
-class StructABI(BaseABI):
+class StructABI(HashableABI):
```

Comment:
Is a struct really selectable in this way?

Reply:
`Struct` has defined `selector()`, so yes? I am not sure I am following the question however. Are you talking about the Ape framework?